workflow/scripts/parallel_vcf.py

```diff
@@ -0,0 +1,50 @@
+import os
+import argparse
+from multiprocessing import Pool
+from pysam import VariantFile
+import tempfile
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument("file_vcf")
+parser.add_argument("command")
+parser.add_argument("--stepsize", "-s", type=int, default=1000000)
+parser.add_argument("--threads", "-t", type=int, default=8)
+parser.add_argument("--tmp", default=None, help="Define the base directory for the tmp files")
+args = parser.parse_args()
+tmp_base = args.tmp
+stepsize = args.stepsize
+
+# create regions
+f = VariantFile(args.file_vcf)
```

Comment:
Use a with statement here, so that you can avoid the close.

Reply:
yes